onnx/reference/ops/aionnxml/op_binarizer.py

```diff
@@ -1,16 +1,10 @@
 # SPDX-License-Identifier: Apache-2.0
 # pylint: disable=R0913,R0914,W0221
 
-import numpy as np
-
+from ....backend.test.case.node.ai_onnx_ml.binarizer import compute_binarizer
```

Comment:
We should define which subfolder must import the other one to avoid circular import. I chose to avoid importing anything from the test case because the implementation was not always complete or even available. That's why I kept subfolders reference and backend completely separated.

Reply:
Can we have the test-cases use the reference-implementation?